components/blitz/resources/omero/cmd/FS.ice

```diff
@@ -185,41 +185,6 @@ module omero {
 
         };
 
-        /**
-         * Request to determine the disk usage of the given objects
-         * and their contents. File-system paths used by multiple objects
-         * are de-duplicated in the total count. Specifying a class is
-         * equivalent to specifying all its instances as objects.
-         *
-         * Permissible classes include:
-         *   ExperimenterGroup, Experimenter, Project, Dataset,
-         *   Folder, Screen, Plate, Well, WellSample,
-         *   Image, Pixels, Annotation, Job, Fileset, OriginalFile.
-         **/
-        class DiskUsage extends Request {
```

Comment:
Over a major version I can't see what the issue there would be but I'm happy to adjust anyway.